src/server/hotkey_collector.cpp

```diff
@@ -122,7 +124,7 @@ void hotkey_collector::capture_msg_data(dsn::message_ex **requests, const int co
             unmarshall(requests[0], thrift_request);
             requests[0]->restore_read();
             key = thrift_request.hash_key;
-            capture_str_data(key.to_string(), thrift_request.kvs.size());
+            capture_blob_data(key, thrift_request.kvs.size());
```

Comment:
This will cause the unit test to fail
```
[==========] Running 1 test from 1 test case.
[----------] Global test environment set-up.
[----------] 1 test from hotkey_collector_test
[ RUN      ] hotkey_collector_test.write_test
F2020-06-17 23:35:39.822 (1592408139822749045 363a) replica.replica0.0101000000000389: hotkey_collector.cpp:121:pegasus_restore_key(): assertion expression: key.length() >= 2 + hash_key_len
F2020-06-17 23:35:39.822 (1592408139822765766 363a) replica.replica0.0101000000000389: hotkey_collector.cpp:121:pegasus_restore_key(): key length must be no less than (2 + hash_key_len)
Aborted (core dumped)
```

Reply:
Yes, I made a mistake. I missed that `capture_blob_data` accepts key as "hashkey+sortkey", and `capture_str_data` accepts key as "hashkey". 

In the above line, I pass a hashkey to `capture_blob_data`. It's illegal. The variable name "key" is misleading.